src/content/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure.mdx

```diff
@@ -708,95 +755,97 @@ Standard roles include:
       </td>
 
       <td>
-        Provides standard access to the platform, including the ability to configure most monitoring tools, but does not include the ability to configure more sensitive settings, including account-level configuration and [synthetic monitor secure credentials](/docs/synthetics/synthetic-monitoring/using-monitors/store-secure-credentials-scripted-browsers-api-tests).
+        Provides access to use most of the platform, but lacks configuration and administration-level abilities (like account-level configuration or [synthetic monitor secure credentials](/docs/synthetics/synthetic-monitoring/using-monitors/store-secure-credentials-scripted-browsers-api-tests)).
       </td>
     </tr>
 
+
     <tr>
       <td>
-        **All product admin**
+        **Billing user**
       </td>
 
       <td>
         Account
       </td>
 
       <td>
-        Provides admin-level access to the platform. This includes all New Relic capabilities with the exception of managing users (**Organization manager** and **Authentication domain manager**) and managing billing-related settings (**Billing user**).
-
-        A [basic user](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-type) is defined as having the **All product admin** role but without access to [Full-Stack Observability](https://newrelic.com/pricing) features.
+        Provides ability to manage subscriptions and billing, and read-only access to the rest of the platform.
       </td>
     </tr>
 
     <tr>
       <td>
-        **Billing user**
+        **Organization manager**
       </td>
 
       <td>
-        Account
+        Organization
       </td>
 
       <td>
-        Provides ability to manage subscriptions and billing, and read-only access to the rest of the platform.
+        Provides the ability to manage [organization](/docs/accounts/accounts-billing/new-relic-one-pricing-users/new-relic-account-structure/-level settings, including organization structure, name, and preferences. Due to our recent switch to the [New Relic One account/user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#user-models), this role currently has few abilities but more will be added over time.
```

Comment:
You're missing a closing parenthesis on that organization link.

Reply:
god bless you